config/config.go

```diff
@@ -15,13 +15,25 @@ type (
 		SessionPath string `toml:"session_path"`
 		ScriptPath  string `toml:"script_path"`
 	}
+	IncludePath struct {
```

Comment:
I don't think "included_paths" is descriptive to the purpose. What do you think of "extended_config" or "additional_config"?

Reply:
I think `extended_config` will be the better fit. And I will make the changes when I have more free time. 🙇